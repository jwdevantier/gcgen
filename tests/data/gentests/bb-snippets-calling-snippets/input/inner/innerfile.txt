# bar snippet calls foo snippet directly, so uses the outer def of foo
# [[start bar
# end]]

# delegates to foo, but uses dynamic lookup, so should use overridden foo def
# [[start baz
# end]]
