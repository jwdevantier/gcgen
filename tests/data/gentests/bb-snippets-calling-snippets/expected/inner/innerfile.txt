# bar snippet calls foo snippet directly, so uses the outer def of foo
# [[start bar
outer bar!
outer foo!
# end]]

# delegates to foo, but uses dynamic lookup, so should use overridden foo def
# [[start baz
outer baz!
inner foo!
# end]]
